storage/pruning/pruningStorer.go

```diff
@@ -202,6 +202,16 @@ func (ps *PruningStorer) Get(key []byte) ([]byte, error) {
 	return v.([]byte), nil
 }
 
+// Close will close PruningStorer
+func (ps *PruningStorer) Close() error {
+	for _, persister := range ps.activePersisters {
+		err := persister.persister.Close()
+		log.LogIfError(err)
```

Comment:
the function never returns an error, it will always return nil

Reply:
agree. maybe define a base error something like `ErrClosingPersisters` and save all errors and return them using `fmt.Errorf`.

Reply:
method return error now if a persister cannot be closed successfully